index.js

```diff
@@ -100,8 +101,29 @@ class Main {
     screen.key(['escape', 'q', 'C-c'], () => process.exit(0));
     // fixes https://github.com/yaronn/blessed-contrib/issues/10
     screen.key(['o', 'O'], () => {
-      const selectedLambdaFunctionName = this.table.rows.items[this.table.rows.selected].data[0];
-      return open(`https://${program.region}.console.aws.amazon.com/lambda/home?region=${program.region}#/functions/${selectedLambdaFunctionName}?tab=configuration`);
+      const selectedLambdaFunctionName = this.table.rows.items[
+        this.table.rows.selected
+      ].data[0];
+      return open(
+        `https://${program.region}.console.aws.amazon.com/lambda/home?region=${program.region}#/functions/${program.stackName}-${selectedLambdaFunctionName}?tab=configuration`,
+      );
+    });
+    screen.key(['d', 'D'], () => {
+      const selectedLambdaFunctionName = this.table.rows.items[
```

Comment:
can we move the contents of this into its own isolated function?
Would also like to see this unit tested :)

Reply:
Moved to a seperate function testing started on seperate branch, made the decision to move setting up testing out of this branch so a timebox can be taken to setup initial testing